packages/react-component-library/src/forms/native/useNativeForm.ts

```diff
@@ -32,11 +32,12 @@ export const useNativeForm = <FormValues>(
     e: React.FormEvent<HTMLInputElement> | SyntheticFormEvent
   ): void => {
     const { name, value } = e.currentTarget
+    const validator = validation[name as keyof Validators<FormValues>]
```

Comment:
Had to use a few type assertions in these form stories unfortunately (open to other suggestions).